commit 48414bfd28fc500d7c10b07594c0779347b9c38e
Author: Davide Cavalca <dcavalca@fb.com>
Date:   Wed Jul 27 14:17:52 2016 -0700

    yummy:patchX:tests--skip-udev-test-if-running-inside-a-chroot.patch

tests: skip udev-test if running inside a chroot


diff --git a/test/udev-test.pl b/test/udev-test.pl
index da0a4e1..35a2668 100755
--- a/test/udev-test.pl
+++ b/test/udev-test.pl
@@ -1535,6 +1535,13 @@ if (!($<==0)) {
         exit($EXIT_TEST_SKIP);
 }
 
+# skip the test when running in a chroot
+system("systemd-detect-virt", "-r", "-q");
+if ($? >> 8 == 0) {
+    print "Running in a chroot, skipping the test.\n";
+    exit($EXIT_TEST_SKIP);
+}
+
 # skip the test when running in a container
 system("systemd-detect-virt", "-c", "-q");
 if ($? >> 8 == 0) {
